.github/workflows/validate-zone.yml

```diff
@@ -0,0 +1,18 @@
+name: Validate zone syntax
+
+on:
+  pull_request:
+  workflow_call:
+
+jobs:
+  named-checkzone:
+    name: "Validate bind9 zone syntax"
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+
```

Comment:
I like them for readability. What's our convention?

Reply:
My consistency is: If it is visible outside the workflow, it's upper case, otherwise it's lower case.

It is very hard to make this really consistent, as GitHub uses upper case for secrets and environment variables, but many other things are lower case. (So even if I change them to lower case, the secrets in the docker role will still be upper case).

Reply:
compare `validate-structure.yaml` and `validate-zone.yaml`

Reply:
Ah... I looked at the wrong consistency. :see_no_evil:

Reply:
There should be consistency now.